### PR TITLE
Update aws_sandbox.md cli examples

### PR DESCRIPTION
**Pull request by cob16**

### CODE REVIEW

File: guides/cloud/aws_sandbox.md [hunk 1]
```diff
@@ -55,8 +55,29 @@ Only employees that are conducting devops pairing interviews will be given acces
...
+
+1. Install [aws-vault](https://github.com/99designs/aws-vault#installing)
+2. Add the following config to your `~/.aws/config` file.
+
+```
+ [profile mt-devops]
+ sso_start_url=https://madetech.awsapps.com/start
+ sso_region=eu-west-2
+ sso_account_id=612473995106
+ sso_role_name=SandboxAdmin
```
Comment (tess-barnes-mt) [inline guides/cloud/aws_sandbox.md:72]: Is this intentionally the same Role as for the sandbox account?

File: guides/cloud/aws_sandbox.md [hunk 2]
```diff
@@ -55,8 +55,29 @@ Only employees that are conducting devops pairing interviews will be given acces
...
+ output=json
+ ```
+
+# Admin actions
+This section is for Sadbox Admin administrators (e.e pepole in the `@sandbox-admins` group)
```
Comment (tess-barnes-mt) [inline guides/cloud/aws_sandbox.md:77]: Suggest that we change Sadbox --> Sandbox and e.e --> i.e in this sentence.

Also, if this is a section specific for SandboxAdmin, should normal use and devops pairing use be including a differently named role in the config examples above?

File: guides/cloud/aws_sandbox.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ Reasons for using this tool can be found [here](https://github.com/99designs/aws
 2. Add the following config to your `~/.aws/config` file.
 
 ```
- [profile madesso]
+ [profile mt-playground]
```
Comment (tess-barnes-mt) [inline guides/cloud/aws_sandbox.md:34]: Should this profile name change be reflected in the example aws-vault command on lines 42 & 44?

**Review by tess-barnes-mt — CHANGES_REQUESTED** [review]

File: guides/cloud/aws_sandbox.md (hunk 1 above)
Comment (cob16) [reply guides/cloud/aws_sandbox.md:72]: yes for simplicity it has the same role name, but in a different accountnt

**Review by tess-barnes-mt — APPROVED** [review]

**Pull request merged**